### PR TITLE
fix(dedicated-ip): sort the ip block FO

### PR DESCRIPTION
**Pull request by sachinrameshn**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/ip-strat`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #.MANAGER-10194 -->
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js [hunk 1]
```diff
@@ -172,6 +172,10 @@ export default class AgoraIpOrderCtrl {
             isIpBlockOffer: maximumQuantity === 1,
           };
         });
+        console.log('bo1', bo);
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:175]: ah ah ah 🥇 :)

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js [hunk 2]
```diff
@@ -172,6 +172,10 @@ export default class AgoraIpOrderCtrl {
...
+        bo.sort((a, b) => a.price.value - b.price.value);
+        console.log('sort bo', bo);
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:177]: :)

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js [hunk 3]
```diff
@@ -157,7 +157,7 @@ export default class AgoraIpOrderCtrl {
   loadPrivateCloudIpOffers(serviceName) {
     return this.IpAgoraOrder.getPrivateCloudIpOffers(serviceName).then(
       (ipOffers) => {
-        this.blockIpOffers = ipOffers.map((offer) => {
+        const bo = ipOffers.map((offer) => {
```
Comment (mohammed-zahaf) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:160]: i guess better naming
```suggestion
        const ipBlockOffers = ipOffers.map((offer) => {
```

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js (hunk 1 above)
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js:175]: **nitpick:** Please remove the console statements.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js (hunk 1 above)
Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js]: my bad removing it

Comment (sachinrameshn) [reply packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.controller.js]: removed it

**Review by mohammed-zahaf — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**